### ISSUE TITLE
[1.12.2][4.0.1] NPE

### ISSUE THREAD

**Issue by FiveEYZ** [M4thG33k/TombManyGraves2#18]
Just updated server to 1.12.2, this crash happen as 1 player got back to his grave

**Version:**
TombManyGraves-1.12-4.0.1
forge-1.12.2-14.23.0.2503-universal

**crash-report/modlist:**
https://gist.github.com/FiveEYZ/d6ee38c44390dfc5ce6c826aa4a4700c

**Comment by M4thG33k** [comment]
Well, I haven't marked it compatible with 1.12.2 yet, but I'll try to take a look into it this weekend. Thanks for bringing it to my attention.

**Comment by M4thG33k** [comment]
Finally getting around to looking at this. Has this issue been repeatable? I haven't been able to reproduce it.

**Comment by FiveEYZ** [comment]
Im using SpongeForge and with that i have plugin GriefPrevention.

I just checked this and if the event for colliding with the grave block is canceled, I noticed that as soon as I went on the grave holding shift the server crashed, when I got online I still gotten the item(s) from the grave, but the grave was still in the world.

The cancellation of the event will occur when a player tries to get a grave in a "claimed area" belonging to another player and is not "trusted" by the other player to alter blocks.

Now this can be fixed on my end overriding the permission for just the grave inside claims, maybe there is nothing that can be done, but at least i know why the server crashed.
